src/main/java/by/exadel/internship/controller/FormController.java

```diff
@@ -0,0 +1,103 @@
+package by.exadel.internship.controller;
+
+import by.exadel.internship.dto.FormDTO;
+import by.exadel.internship.dto.InterviewDTO;
+import by.exadel.internship.dto.enums.EnglishLevel;
+import by.exadel.internship.dto.enums.FormStatus;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
Will you include "api" to all mappings? If yes, better to add it as a context path to your configuration. 
Here is the good reference https://www.baeldung.com/spring-boot-context-path#overview

Reply:
Done!